src/inference/tests/functional/core_threading.cpp

```diff
@@ -63,7 +63,7 @@ class IECoreThreadingTests : public ::testing::Test {
         try {
             auto extension = std::make_shared<InferenceEngine::Extension>(
                 ov::util::make_plugin_library_name(ov::test::utils::getExecutableDirectory(),
-                                                   std::string("template_extension") + OV_BUILD_POSTFIX));
+                                                   std::string("openvino_template_extension") + OV_BUILD_POSTFIX));
```

Comment:
`IECoreThreadingTests.ReadNetwork` failed because they don't work with new extensions.
I suppose we can check whether the same test exists in API 2.0 already. If yes, we can safely drop this test